src/main/java/lion/like/galleryapi/controller/GalleryController.java

```diff
@@ -0,0 +1,61 @@
+package lion.like.galleryapi.controller;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.v3.oas.annotations.Operation;
+import lion.like.galleryapi.domain.Comment;
+import lion.like.galleryapi.domain.Image;
+import lion.like.galleryapi.dto.CommentRequest;
+import lion.like.galleryapi.dto.CommentResponse;
+import lion.like.galleryapi.service.GalleryService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Api(tags = "GalleryController")
+@RestController
+public class GalleryController {
+
+    private final GalleryService galleryService;
+
+    public GalleryController(GalleryService galleryService) {
+        this.galleryService = galleryService;
+    }
+
+    @Operation(summary="특정 이미지 댓글 생성", description="특정 이미지의 댓글을 생성합니다.")
+    @ApiResponse(code = 200, message="ok")
```

Comment:
```suggestion
    @Operation(summary = "특정 이미지 댓글 생성", description = "특정 이미지의 댓글을 생성합니다.")
    @ApiResponse(code = 200, message = "ok")
```

코드 스타일은 되도록 통일하는게 좋은데요, 좋은 도구를 찾아보는 것도 도움 될 것 같아요 😄